odoo_backup_sh/doc/index.rst

```diff
@@ -75,3 +75,14 @@ Usage
 RESULT: Backup is restored in one click without any additional manipulations such as "downloading-uploading process".
 
 *The database restore may take some time before being ready*.
+
+Downloading backups
+===================
+
+* Proceed to the Backup list: ``/web/database/backups``
+* Click on ``Download`` button and wait until download completes
+* Backup is encrypted (it has `.enc` extension), extract it using `gpg` utility. As example:
+
+```
+gpg --output OUTPUT_FILENAME.zip --decrypt INPUT_FILENAME.zip.enc
+```
```

Comment:
Add a note like "Type the password when it will be prompted"

Reply:
fixed